modules/ossm-federation-create-export.adoc

```diff
@@ -11,6 +11,11 @@ You create an `ExportedServiceSet` resource to explicitly declare the services t
 
 Services are exported as `<export-name>.<export-namespace>.svc.<ServiceMeshPeer.name>-exports.local` and will automatically route to the target service.  This is the name by which the exported service is known in the exporting mesh. When the ingress gateway receives a request destined for this name, it will be routed to the actual service being exported. For example, if a service named `ratings.red-mesh-bookinfo` is exported to `green-mesh` as `ratings.bookinfo`, the service will be exported under the name `ratings.bookinfo.svc.green-mesh-exports.local`, and traffic received by the ingress gateway for that hostname will be routed to the `ratings.red-mesh-bookinfo` service.
```

Comment:
This comment is out of scope for this PR, but I noticed throughout the `ossm-federation` assembly, there are a lot of user-replaceable values that aren't following [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values) (e.g., all lowercase, `_` between multiple words, always marked up with backticks).

Would recommend doing a larger sweep in a follow-up PR for this types of issues.
 
```suggestion
Services are exported as `<export_name>.<export_namespace>.svc.<servicemeshpeer_name>-exports.local` and will automatically route to the target service.  This is the name by which the exported service is known in the exporting mesh. When the ingress gateway receives a request destined for this name, it will be routed to the actual service being exported. For example, if a service named `ratings.red-mesh-bookinfo` is exported to `green-mesh` as `ratings.bookinfo`, the service will be exported under the name `ratings.bookinfo.svc.green-mesh-exports.local`, and traffic received by the ingress gateway for that hostname will be routed to the `ratings.red-mesh-bookinfo` service.
```
(Just an example, not sure if `.name` is special for `ServiceMeshPeer` objects, like if it should be `<servicemeshpeer>.name`)

Reply:
Created OSSM-5966 and assigned it to myself so this is not lost.